mltsp/Flask/flask_app.py

```diff
@@ -1031,19 +1034,22 @@ def add_featureset(
     return new_featset_key
 
 
-def add_model(
-        featureset_name, featureset_key, model_type, projkey, pid,
-        meta_feats=False):
+def add_model(model_name, featureset_name, featureset_key, model_type,
+              model_params, projkey, pid, meta_feats=False):
     """Add a new entry to the rethinkDB 'models' table.
 
     Parameters
     ----------
-    name : str
-        New feature set name.
+    model_name : str
+        New model name.
+    featureset_name : str
+        Name of associated feature set.
     featureset_key : str
         RethinkDB key/ID of associated feature set.
     model_type : str
-        Abbreviation of model/classifier type (e.g. "RF").
+        Abbreviation of model/classifier type (e.g. "RFC").
+    model_params : dict
+        Dictionary describing model parameters.
```

Comment:
Ah, I see what this is now.  Perhaps just the very slightly more verbose: "Dictionary containing names of scikit-learn model parameters and their values.  Values can be strings, ints and ???." would have helped me.